pyproject.toml

```diff
@@ -50,6 +50,7 @@ ignore = [
     "SIM102",
     "SIM105",
     "RUF012", # We may want to enable "Mutable class attributes should be annotated with `typing.ClassVar`"
+    "COM812", # Not recommended because we also use ruff as a formatter (see https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules)
```

Comment:
Could we move this change to an other PR, as it is unrelated with the other changes about SQLAlchemy and database.

To be honest, I would be in favor of keeping this rule, which requires trailing comma. In practice it seems to work well with the formater.

Reply:
> Could we move this change to an other PR, as it is unrelated with the other changes about SQLAlchemy and database.
Sure, I added this change here because I consider this pull request as several individual improvement of the code but I will move it to a different PR as it is more controversial than I expected.